test/non_hardware/test_flasherpyocd.py

```diff
@@ -109,6 +109,11 @@ def test_cy8cproto_062_4343w(self):
         self.assertEqual(PyOCDMap.platform('CY8CPROTO_062_4343W'), 'cy8c6xxA')
         self.assertEqual(PyOCDMap.pack('CY8CPROTO_062_4343W'), None)
 
+    def test_k64f(self):
```

Comment:
Could we also capitalize the letters here, for consistency?

Reply:
Test case names are kept consistent by not using upper case :)